### DISCUSSION TITLE
Categorization of books library by tags

### DISCUSSION THREAD

**Discussion by sagashiman** [Goodwy/PlayBooks#65] [Ideas]
As of now there is no way to some how sort books by categories. There is only list of books on main screen in two categories: "Current" and "Not started yet". It would be nice to have some feature which would allow user to manage his library.

**Comment by tobias-p** [comment]
I want to add that this would be great. 

Having audiobooks for children (like a lot of the same franchise, 50+ episodes) and no possibility to sort them or categorize them will drive me crazy. Please add this feature, to help us categorize the audiobooks 🥺 🙏
